src/util/options.schema.json

```diff
@@ -1697,6 +1697,12 @@
           "type": "boolean",
           "default": true
         },
+        "float": {
+          "title": "warn.float",
+          "description": "float warnings",
+          "type": "boolean",
+          "default": true
```

Comment:
i guess our new options should be defaulted to `false`, so the behaviour of old examples does not change